test/javascripts/controllers/create-account-test.js.es6

```diff
@@ -84,3 +84,19 @@ QUnit.test("passwordValidation", function(assert) {
   testInvalidPassword("porkchops", I18n.t("user.password.same_as_username"));
   testInvalidPassword("pork@chops.com", I18n.t("user.password.same_as_email"));
 });
+
+QUnit.test("authProviderDisplayName", function(assert) {
+  const subject = this.subject;
```

Comment:
I feel like this line is not needed and we can just call this.subject at next line, given we don’t use it at other places

Reply:
Done